css/styles.css

```diff
@@ -0,0 +1,104 @@
+/* Table of contents
+__________________________________
+-General Content
+-Typography
+-Header & Navigation
+-Images & Media
+-Footer
+-Media Queries
+__________________________________*/
+
+/* -General Content
+__________________________________*/
+.main{padding-top:3%;}
+.row{padding: 2% 0;}
+.blue{background-color: #9dc6d8;}
+.white{background-color: #fff;}
+.gray{background-color: #c6cbcc;}
```

Comment:
When specifying a color make sure you state what the class is apply the color to.
If I used gray here with the intent of it changing the text it wouldn't work.

Use .gray-text or .gray-bg